### PR TITLE
add infos

### PR DESCRIPTION
**Pull request by dapparu**
- jets info
- E / p
- mini iso with muon contrib

### CODE REVIEW

File: Analyzer/interface/CommonFunction.h [hunk 1]
```diff
@@ -1174,7 +1174,7 @@ bool clusterCleaning(std::vector<int> ampls, int crosstalkInv = 0, uint8_t* exit
       MaxOnEnd = true;
     }
   }
-  // Si une seule strip touch�e
+  // Si une seule strip touchée
```
Comment (tvami) [inline Analyzer/interface/CommonFunction.h:1177]: Can you actually translate that to english please? :)

File: Analyzer/interface/Regions.h [hunk 2]
```diff
@@ -10,10 +10,9 @@
 #include "TDirectory.h"
 
 
-TRandom3* RNG = new TRandom3();
 
-float K = 1;
-float C = 1;
+float K = 2.30;
+float C = 3.17;
```
Comment (tvami) [inline Analyzer/interface/Regions.h:15]: Why does this matter? It should be not hardcoded by modified from the config, right?

File: Analyzer/interface/CommonFunction.h (hunk 1 above)
Comment (dapparu) [reply Analyzer/interface/CommonFunction.h:1177]: sure!

Comment (dapparu) [reply Analyzer/interface/CommonFunction.h:1177]: I'll translate all sentences to english

File: Analyzer/interface/Regions.h (hunk 2 above)
Comment (dapparu) [reply Analyzer/interface/Regions.h:15]: It should, forget of my side

File: Analyzer/plugins/Analyzer.cc [hunk 3]
```diff
@@ -1109,8 +1148,35 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
...
+            if(pf_isPhotonForIdx) track_PFMiniIso_sumPhotonPt+=pt;
+            // muon iso
+            if(pf_isMuonForIdx) track_PFMiniIso_sumMuonPt+=pt;
+        }
       }//end loop PFCandidates
     }
+    
+  // Calculate PF mini relative isolation
+  float miniRelIsoAll = (track_PFMiniIso_sumCharHadPt + std::max(0.0, track_PFMiniIso_sumNeutHadPt + track_PFMiniIso_sumPhotonPt - 0.5* track_PFMiniIso_sumPUPt))/track->pt();
+  float miniRelIsoAll_wMuon = (track_PFMiniIso_sumMuonPt + track_PFMiniIso_sumCharHadPt + std::max(0.0, track_PFMiniIso_sumNeutHadPt + track_PFMiniIso_sumPhotonPt - 0.5* track_PFMiniIso_sumPUPt))/track->pt();
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.cc:1177]: what about the electron?

File: Analyzer/plugins/Analyzer.cc [hunk 4]
```diff
@@ -1533,7 +1599,7 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
                                                 CutTOF_Flip_);
     } else {
       // Preselection not passed, skipping it
-      continue;
+      //continue; // NOOOOOO !! DONT SKIP IT ! SKIP CAUSES PROBLEM FOR THE TUPLE
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.cc:1602]: well but I need to skip it for the histo based code :D not sure what we should do...

Comment (dapparu) [reply Analyzer/plugins/Analyzer.cc:1602]: Really? I'm not sure because you fill the histograms in the PassPreselection function

Comment (dapparu) [reply Analyzer/plugins/Analyzer.cc:1602]: Oh ok it's for the ones passing preselection...

File: Analyzer/plugins/Analyzer.cc (hunk 3 above)
Comment (tvami) [reply Analyzer/plugins/Analyzer.cc:1177]: right but what's the motivation to not include them? maybe something to discuss tomorrow

File: Analyzer/plugins/Analyzer.cc (hunk 4 above)
Comment (tvami) [reply Analyzer/plugins/Analyzer.cc:1602]: I fixed this in https://github.com/CMS-HSCP/SUSYBSMAnalysis-HSCP/pull/43

**Pull request closed without merging**